src/GitVersion.Configuration/ConfigurationFileLocator.cs

```diff
@@ -30,11 +33,17 @@ public void Verify(string? workingDirectory, string? projectRootDirectory)
     public string? GetConfigurationFile(string? directory)
     {
         if (directory is null) return null;
-        string?[] candidates = [this.ConfigurationFile, DefaultFileName, DefaultAlternativeFileName];
-        var candidatePaths =
-            from candidate in candidates
-            where !candidate.IsNullOrWhiteSpace()
-            select PathHelper.Combine(directory, candidate);
+        var candidateList = new List<string>(PossibleConfigFileNames);
```

Comment:
I'd prefer to keep the old syntax

```suggestion
        string?[] candidates = [this.ConfigurationFile, ..SupportedConfigFileNames];
        var candidatePaths =
            from candidate in candidates
            where !candidate.IsNullOrWhiteSpace()
            select PathHelper.Combine(directory, candidate);
```